business/authentication/token_auth_controller.go

```diff
@@ -63,18 +64,21 @@ type UserSessionData struct {
 	//
 	// example: zI1NiIsIsR5cCI6IkpXVCJ9.ezJ1c2VybmFtZSI6ImFkbWluIiwiZXhwIjoxNTI5NTIzNjU0fQ.PPZvRGnR6VA4v7FmgSfQcGQr-VD
 	// required: true
-	Token string
+	Token string `json:"-"`
```

Comment:
Yes. I originally didn't put any `json` annotation with the intention of not being sent to the browser. However, here I saw the token was being sent. So, I'm removing it. The caller function *do* uses the `Token` and puts it in the `context()` so that it can be used later, but it shouldn't be travelling over the network. The UI is currently not using the token...